packages/plugin-essentials/sources/suggestUtils.ts

```diff
@@ -335,3 +338,17 @@ export async function fetchDescriptorFrom(ident: Ident, range: string, {project,
   return structUtils.makeDescriptor(bestLocator, structUtils.makeRange({protocol, source, params, selector}));
 }
 
+export async function makeTemporaryCache() {
```

Comment:
No need to export it since it's not a suggest util - I'd also prefer to use the "cleanup on return pattern", by making `makeTemporaryCache` accept a callback that takes the temporary cache as parameter (cf the mktemp signature for an example).